html-forms/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+
+    <h1>Coding Form!</h1>
+    <form action=" " method="post">
+      <p>Email <br />
```

Comment:
this should be a label, and make sure that you avoid using `br` tags. breaking input elements on to separate lines should be done using `divs`